ONSdigital/eq-survey-runner
# Fix for piped answers in repeating groups

### What is the context of this PR?
This PR is to fix an issue where answers within repeating groups could not be referenced within piped strings. As a side effect of this fix I have also removed the answer alias functionality which provided little value other than to allow piped answer values to be referenced with underscored attribute names.

### How to review 
- Run test_repeating_household schema and assert that the new confirmation block displays the correct answers that we added within the repeating group
- Check all other schemas that used piped values to be sure that they still operate correctly

PRs for publisher and validator also incoming.
